### PR TITLE
Adapt import images requests for views in studio

### PR DESCRIPTION
**Pull request by qtreh**
- Add org argument for cmd add_images
- Stop using commit for studio requests

Tested with a local studio web endpoint
=> Small changes in vesta needed for it to work: https://github.com/Deepomatic/vesta/pull/3084

### CODE REVIEW

File: deepomatic/cli/cmds/studio_helpers/file.py [hunk 1]
```diff
@@ -153,11 +153,11 @@ def fill_queue(self, files, dataset_name, commit_pk):
         self.flush_batch(url, batch)
         return total_files
 
-    def post_files(self, dataset_name, files):
+    def post_files(self, org_slug, dataset_name, files):
         # Retrieve endpoint
         try:
-            ret = self._helper.get('datasets/' + dataset_name + '/')
+            request = 'orgs/{}/projects/{}/'.format(org_slug, dataset_name)
```
Comment (maingoh) [inline deepomatic/cli/cmds/studio_helpers/file.py:159]: Is the org mandatory now @Alexis-Jacob ?

**Review by maingoh — COMMENTED** [review]
Does the upload format change as well ?
The tests don't seem to succeed

File: deepomatic/cli/cmds/studio_helpers/file.py (hunk 1 above)
Comment (Alexis-Jacob) [reply deepomatic/cli/cmds/studio_helpers/file.py:159]: yes, the org is the root of all api calls now

**Comment by qtreh** [comment]
> Does the upload format change as well ?
> The tests don't seem to succeed

@maingoh The error is: resource = 'https://studio.deepomatic.com/api/orgs/travis-deepocli/projects/deepocli-feedback-test-detection/' is not found. The old API (current studio) works with /datasets/, the new one (staging, dev) with /projects/.

 Jenkins uses studio which has not been switched to the new API yet, we need to wait for a release. Once the release is done, this test should work, and we'll be able to merge.

File: deepomatic/cli/cmds/studio.py [hunk 2]
```diff
@@ -128,6 +129,7 @@ def setup_cmd_line_subparser(studio_subparser):
     # Define studio group for add_images
     group = add_images_parser.add_argument_group('studio arguments')
     group.add_argument('-d', '--dataset', required=True, help="Deepomatic Studio dataset name.", type=str)
```
Comment (maingoh) [inline deepomatic/cli/cmds/studio.py:131]: Since the compatiblity is broken, maybe rename `dataset` into `project` ?

**Review by maingoh — APPROVED** [review]
LGTM

**Review by Alexis-Jacob — APPROVED** [review]

**Pull request merged**